server/tests/testutils/factory_alchemy.py

```diff
@@ -0,0 +1,73 @@
+from typing import Any, Optional
+
+from factory.base import BaseFactory, FactoryMetaClass, resolve_attribute
+from sqlalchemy.orm.attributes import InstrumentedAttribute
+from sqlalchemy.orm.decl_api import DeclarativeMeta
+
+
+class SQLAlchemyModelFactoryMetaClass(FactoryMetaClass):
+    class UnhandledModelAttr(Exception):
+        pass
```

Comment:
Can you please document how this works, because metaclasses are something not obvious at all, and if something breaks, others will have a hard time debugging the reason.

Reply:
added documentation